lib/api/build.rs

```diff
@@ -27,8 +27,12 @@ fn main() -> std::io::Result<()> {
     // Fetch git commit ID and pass it to the compiler
     match Command::new("git").args(["rev-parse", "HEAD"]).output() {
         Ok(output) => {
-            let git_commit_id = str::from_utf8(&output.stdout).unwrap().trim();
-            println!("cargo:rustc-env=GIT_COMMIT_ID={git_commit_id}");
+            if output.status.success() {
+                let git_commit_id = str::from_utf8(&output.stdout).unwrap().trim();
+                println!("cargo:rustc-env=GIT_COMMIT_ID={git_commit_id}");
+            } else {
+                println!("cargo:warning=current git commit hash could not be determined");
+            }
```

Comment:
@timvisee the warning didn't work previously because we need to check success status of the subprocess as well.